scripts/core/get-superdesk-api-implementation.tsx

```diff
@@ -197,6 +198,9 @@ export function getSuperdeskApiImplementation(
             },
         },
         state: applicationState,
+        instance: {
+            config: JSON.parse(JSON.stringify(config || {})), // deep clone of config object
```

Comment:
imo it would make sense to have it in some utils `utils.cloneDeep` or use lodash for it

Reply:
I'm actually gonna pass the object directly because there's no advantage of cloning here and also the method I used is not reliable because JSON doesn't support values like Date, functions, etc